jekyll-assets/_includes/head.html

```diff
@@ -2,6 +2,7 @@
 <meta http-equiv="X-UA-Compatible" content="IE=edge">
 <meta name="viewport" content="width=device-width, initial-scale=1">
 <title>{{ page.title }}</title>
+<meta name="description" content="Raspberry Pi documentation">
 <meta property="og:title" content="{{ page.title }}">
 <meta property="og:description" content="The official documentation for Raspberry Pi computers and microcontrollers">
 <meta property="og:url" content="https://www.raspberrypi.com{{ page.url | relative_url }}">
```

Comment:
To me it makes sense to add the description to `_config.yml` which appears now three times here. What about adding it [here](https://github.com/raspberrypi/documentation/blob/267bfeb/_config.yml#L17-L18) (instead of doubling the title) and then using `{{ page.description }}` or `{{ site.description }}` for HTML, Open Graph, and Twitter descriptions?